src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+
+const daysWithLowADiscount = 3;
+const daysWithLargeADiscount = 7;
+const rentOfEveryDay = 40;
+const lowDiscount = 20;
+const largeDiscount = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= daysWithLowADiscount && days < daysWithLargeADiscount) {
+    return days * rentOfEveryDay - lowDiscount;
+  }
+
+  if (days >= daysWithLargeADiscount) {
+    return days * rentOfEveryDay - largeDiscount;
+  }
+
+  return days * rentOfEveryDay;
```

Comment:
It's good practice to handle the default case last, as you've done here, where no discounts apply.